src/server/services/wfs/qgswfstransaction.cpp

```diff
@@ -689,6 +716,14 @@ namespace QgsWfs
         action.errorMsg = QStringLiteral( "%1 '%2'" ).arg( ex.message() ).arg( typeName );
         continue;
       }
+
+      if ( featureList.size() == 0 )
```

Comment:
```
if ( featureList.empty() ) 
{ 
   ...
}
```

Reply:
@pblottiere are you aware of the "propose change" button in the review / comment editor? That will make the fixups very easy to apply.

Reply:
```suggestion
      if ( featureList.empty() )
```

Reply:
Hi @m-kuhn,

> are you aware of the "propose change" button in the review / comment editor? That will make the fixups very easy to apply.

Nope, thanks for the tip!